<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Newdiag</class>
 <widget class="QDialog" name="Newdiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Projekt típusa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="projecttype">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>C++</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debian Package</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Git client</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Add file</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mégse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createprojectbutton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Létrehozás</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Project tulajdonságai</string>
   </property>
   <widget class="QLineEdit" name="project_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="project_path">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Project neve</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Project Helye</string>
    </property>
   </widget>
   <widget class="QPushButton" name="project_path_set">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>130</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hely kiválasztása</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="project_url">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Project URL-je</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
